library.wdl

```diff
@@ -57,7 +57,7 @@ workflow Library {
         input:
             bamFile = markdupBamFile,
             outputDir = libraryDir,
-            bamName =  sample.id + "-" + library.id + ".bqsr",
+            bamName =  sample.id + "-" + library.id,
```

Comment:
Why change this?

Reply:
This is changed in GATK-preprocess upstream